lite/README.md

```diff
@@ -1 +1,141 @@
-# valor-lite: Compute classification, object detection, and segmentation metrics locally.
+# valor-lite: Fast, local machine learning evaluation.
+
+Valor-lite is a lightweight, numpy-based library designed for fast and seamless evaluation of machine learning models. It is optimized for environments where quick, responsive evaluations are essential, whether as part of a larger service or embedded within user-facing tools. By operating without the latency of external systems, Valor-lite ensures an efficient, smooth experience for users.
+
+Valor, the broader platform, acts as a centralized evaluation store that simplifies the measurement, exploration, and ranking of model performance. This powerful tool helps data scientists and engineers continuously assess and refine their machine learning pipelines, enabling them to make data-driven decisions for ongoing improvement.
```

Comment:
can you add a note that the Valor service uses valor-lite to do its computations, and that we published valor-lite separately in case users want to evaluate performance without managing that separate service?

nit: some of these other benefits could live in the repo README.md, rather than in `lite/README.md`; talking about them both here seems confusing.

Reply:
trimmed it!